ui/components/list-row.js

```diff
@@ -0,0 +1,22 @@
+import { CheckIcon } from '@heroicons/react/solid'
+
+export default function ListRow({ item, selected }) {
```

Comment:
Last improvements for this: `ComboboxItem` might be a better name given it always shows up in a combo box

Would be better to decouple this a bit more by changing the props to `title`, `subtitle`, `selected=true|false` to render it as selected or not?